soc/src/aspeed/ast2500/reg.rs

```diff
@@ -0,0 +1,22 @@
+const CONFIG_DRAM_ECC_SIZE: u32 = 0x10000000;
+const BASE: u32 = 0x1E60_0000;
```

Comment:
It might be useful to use tock's register crate. See the pl011.rs for an example.